alica_dummy_proxy/include/communication/AlicaDummyCommunication.h

```diff
@@ -11,15 +11,15 @@ struct Queues;
 class AlicaDummyCommunication : public alica::IAlicaCommunication
 {
 public:
-    AlicaDummyCommunication(alica::AlicaEngine* ae);
+    AlicaDummyCommunication(alica::AlicaCommunicationHandlers& callbacks);
```

Comment:
`const alica::AlicaCommunicationHandlers&`
similarly need to change in other places